actions/monitoring/verify_installation_monitoring/action.yaml

```diff
@@ -0,0 +1,26 @@
+name: 'Verify Monitoring Installation Action'
+description: 'A GitHub Action to verify tests, get logs from pods'
+inputs:
+  namespace:
+    description: 'namespace where service is installed'
+    required: true
+    default: monitoring
+  max_attempts:
+    description: 'maximum number of attempts to verify that pod status is successful'
+    required: true
+    default: 40
+  timeout:
+    description: 'timeout between attempts'
+    required: true
+    default: '10s'
+
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Run helm_install action
+      uses: ./qubership-test-pipelines/actions/shared/verify_installation
```

Comment:
The action is `verify_installation`. What does it do? Does it verify only? Does it install as well? It would be nice if `name` was consistent with the performed steps.

Reply:
This action only verifies installation. Name fixed